lms/static/scripts/frontend_apps/components/Dialog.js

```diff
@@ -60,23 +52,18 @@ export default function Dialog({
       // dialog. See resources above.
       rootEl.current.focus();
     }
-
     // We only want to run this effect once when the dialog is mounted.
     //
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
   return (
-    // FIXME-A11Y
-    // eslint-disable-next-line jsx-a11y/no-noninteractive-element-interactions
     <div
-      role="dialog"
-      aria-labelledby="Dialog__title"
-      onKeyDown={handleKey}
-      // FIXME-A11Y
-      // eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex
-      tabIndex="0"
+      role={role}
+      aria-labelledby={dialogId}
+      aria-modal="true"
       ref={rootEl}
+      tabIndex="-1"
```

Comment:
Decided to change this to -1 so it would not be something you can arrive at by keyboard. We really just use focus here when it opens as a fallback when a button or child element is not specified.